javascript-carousel/main.js

```diff
@@ -0,0 +1,80 @@
+const $imageRow = document.querySelector('#image-row');
+const $circleRow = document.querySelector('#circle-list');
+const $circleList = $circleRow.children;
+const $pokemonImage = document.querySelector('#pokemon-img');
+
+var intervalID = setInterval(scrollRight, 3000);
+
+$imageRow.addEventListener('click', function (event) {
+  if (event.target.id !== 'left' && event.target.id !== 'right') return;
+  let currentImage = currentActive();
+  if (event.target.id === 'right') {
+    scrollRight();
+    resetScroll(intervalID);
+    return;
+  }
+  if (event.target.id === 'left') {
+    currentImage--;
+    if (currentImage === -1) {
+      currentImage = 4;
+    }
+  }
```

Comment:
Using a `scrollRight` function seems like a good plan. It might also make sense to use a `scrollLeft` function.

Reply:
ScrollLeft function added and listeners simplified